health-services/individual/src/main/resources/db/migration/main/V20260712120000__add_indexing_for_individual_mobilenumber_ddl.sql

```
@@ -0,0 +1,7 @@
+-- Adds a btree index on INDIVIDUAL.mobileNumber.
+-- Mobile-number searches (validation, create pre-checks, HRMS user validation) previously did a
+-- full sequential scan of the individual table because this column was never indexed
+-- (V20230314122200 indexed givenName/familyName/gender/etc. but not mobileNumber).
+-- The column stores deterministic ciphertext (keyId|ciphertext), so exact-match IN lookups
+-- use this index. Verified live: Index Scan, ~0.17 ms vs a full ~490k-row seq scan.
+CREATE INDEX IF NOT EXISTS idx_individual_mobilenumber ON INDIVIDUAL (mobileNumber);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)